Quickfix

TadJohnson00 opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
Everything from the Arsenal Expansion PR (https://github.com/DeltaV-Station/Delta-v/pull/189), plus some balances to cargo pricing, sprite updates, a few other things

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
Arsenal Expansion PR: adding new weapons for mercenaries, potential addition to organic gameplay too. Rebalance of prices was necessary, rifles and Universals were far too cheap comparatively.
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
N/A
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->
I think it's pretty self-explanatory and I'm mad tired, if you want me to show stuff there's really not much to show but ask and I'll try to grab a screenie
![268429206-16600fe5-2808-44fe-935d-b9a7f0431d00](https://github.com/DeltaV-Station/Delta-v-rebase/assets/120885811/9b30747a-1eb4-422c-9e2b-4cbe146bbdcd)
That's from the Arsenal Expansion PR
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

Colin-Tel commented:
It's kind of dumb there aren't any "emergency" air/n2 tanks, because that would really come in handy here.

DangerRevoltion commented:
> It's kind of dumb there aren't any "emergency" air/n2 tanks, because that would really come in handy here.

I am actually working on that now; I've got a working emergency N2 tank, I'm just recolouring the emergency O2 tank sprites now :)

DangerRevoltion commented on Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml:
```diff
@@ -93,6 +93,33 @@
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 100
+# I've tried to look into it and I can't see why setting the emergency nitrogen tank to
```
still looking into this.

DangerRevoltion commented on Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml:
```diff
@@ -93,6 +93,33 @@
+# type: clothing
+# type: gas tank similarly to the emergency o2 tank breaks it.
+# this is the cleanest I could do - Danger!Revolution!
+
```
still looking into this.

DangerRevoltion commented:
Does this not cause issues by randomising the viability of emergency lockers?
My interpretation of your suggestion was to make
- emergency oxygen tank
- oxygen tank
- emergency nitrogen tank
- nitrogen tank
all mutually exclusive in the loot table?
Hence: you, as a human, could now walk up to an emergency locker and potentially find a nitrogen tank in there; creating a new problem?
or: a slime could walk up to an emergency locker and STILL not find Nitrogen despite the fix?

DebugOk commented:
> Does this not cause issues by randomising the viability of emergency lockers?
> My interpretation of your suggestion was to make
> - emergency oxygen tank
> - oxygen tank
> - emergency nitrogen tank
> - nitrogen tank
> all mutually exclusive in the loot table?
> Hence: you, as a human, could now walk up to an emergency locker and potentially find a nitrogen tank in there; creating a new problem?
> or: a slime could walk up to an emergency locker and STILL not find Nitrogen despite the fix?

I can see this getting abused, but eh, should be fine as is then. It'd be less of an issue of slimes started with the proper tank

DebugOk commented:
You dont need to keep merging master

TadJohnson00 committed:
Arsenal Expansion

Basically my Arsenal Expansion PR on oldrepo now on rebase. Balances some cargo prices, fixes a few item appearances and descriptions, adds new weapons, cleans sprites... yada yada.

TadJohnson00 committed:
Merge branch 'DeltaV-Station:master' into quickfix

TadJohnson00 committed:
Update pistols.yml

Per Floofers' review

Signed-off-by: Tad "Taddy" Johnson <120885811+TadJohnson00@users.noreply.github.com>

TadJohnson00 committed:
Update e_sword.yml

Per Floofers' review

Signed-off-by: Tad "Taddy" Johnson <120885811+TadJohnson00@users.noreply.github.com>

TadJohnson00 committed:
Update posters.yml

Per Floofers' review

Signed-off-by: Tad "Taddy" Johnson <120885811+TadJohnson00@users.noreply.github.com>

TadJohnson00 committed:
Update magnum.yml

Per Floofers' review

Signed-off-by: Tad "Taddy" Johnson <120885811+TadJohnson00@users.noreply.github.com>

TadJohnson00 committed:
Fixed tags.yml

Per Floofers review :3

TadJohnson00 committed:
Fix sprite references

I think

TadJohnson00 committed:
Merge branch 'DeltaV-Station:master' into quickfix

Colin-Tel committed:
Update Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

Signed-off-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>

Colin-Tel committed:
Update Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

Signed-off-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>

TadJohnson00 committed:
Update e_sword.yml

raaagh

Signed-off-by: Tad "Taddy" Johnson <120885811+TadJohnson00@users.noreply.github.com>

TadJohnson00 committed:
Merge branch 'DeltaV-Station:master' into quickfix

TadJohnson00 committed:
Even quicker fixes

so quick, such wow

TadJohnson00 committed:
Update cargo_security.yml

Update price

TadJohnson00 committed:
Merge branch 'master' into quickfix

TadJohnson00 committed:
Merge branch 'DeltaV-Station:master' into quickfix
